### PR TITLE
Fix #1271 - optional log on production

### LINKED ISSUE #1271: Disable log on production
Hi,

I noticed  my website once builded still outputs the following log in the console : `React Static: Templates Reloaded`. Where does this comes from? I think it should not be in production.

Thanks!

### PR DESCRIPTION
**Pull request by fanich37**
## Description

Possible to choose whether log on production is `on` or `off`

## Changes/Tasks

- [x] Changed code

## Motivation and Context

It closes #1271 

## Types of changes

- [ ] Refactoring/add tests (refactoring or adding test which isn't a fix or add a feature)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have updated the documentation accordingly
- [x] I have updated the CHANGELOG with a summary of my changes
- [ ] My changes have tests around them

### CODE REVIEW

**Comment by fanich37** [comment]
> Can you please invert this logic?
> 
> Instead of `productionLog`: true, it should be `silent`: true to hide the message, to not change the default, current behaviour.

Done.

File: docs/config.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ A `static.config.js` file is optional, but recommended at your project root to u
 - [disablePreload](#disablePreload)
 - [babelExcludes](#babelExcludes)
 - [productionSourceMaps](#productionSourceMaps)
+- [productionLog](#productionLog)
```
Comment (SleeplessByte) [inline docs/config.md]: Should probably change to silent

**Comment by SleeplessByte** [comment]
You may also add a line to the top section in the change log, linking to this pr!

File: docs/config.md (hunk 1 above)
Comment (fanich37) [reply docs/config.md]: Ooops... My bad. Fixed in the last commit.

**Comment by fanich37** [comment]
> You may also add a line to the top section in the change log, linking to this pr!

Done!

**Pull request merged**